app/enterprise/1.5.x/developer-portal/configuration/authentication/sessions.md

```diff
@@ -82,3 +87,24 @@ portal_session_conf = {
     "cookie_secure":false
 }
 ```
+
+### Domains
+
+The dev portal `portal_api_url` and the dev
+portal api `portal_gui_host` must share a domain or subdomain. The following
```

Comment:
Isn't this backwards? 

```
The dev portal `portal_api_url` and the dev portal api `portal_gui_host`
```

GUI host is for the dev portal, and the API URL is for the API.

Reply:
Doh!

Reply:
Fixed in commit a736660